titan/model.py

```diff
@@ -436,6 +436,8 @@ def enter(self):
                         len(self.exits[strategy.exit_class]) * entrance.prob
                     )
 
+                # all agent characterstics depend on location and race, so use
+                # those to figure out characteristics
```

Comment:
I thought the reason we did this was to ensure the population distribution by race and location stayed consistent with params?